test/bootstrap_form_group_test.rb

```diff
@@ -237,14 +237,14 @@ class BootstrapFormGroupTest < ActionView::TestCase
 
   test "form_group creates a valid structure and allows arbitrary html to be added via a block" do
     output = @horizontal_builder.form_group :nil, label: { text: 'Foo' } do
-      %{<p class="form-control-static">Bar</p>}.html_safe
+      %{<input class="form-control-plaintext" value="Bar">}.html_safe
```

Comment:
We like to keep PRs very focused if possible. In this case we should only be changing the test cases that actually include a call to `static_control`. Sorry if my ruminations in the issue confused things. Could you please revert this file to the existing tests? Thanks in advance.

Reply:
Okay, I made this change because the `form-control-static` class no longer exists and I was trying to purge the project of any reference to it. We could do it in a separate PR, but it seemed related so I made the change in this one.

Reply:
I'm OK with leaving these changes in. I like the idea of removing all references to `form-control-static` in this PR.